resources/META-INF/plugin.xml

```diff
@@ -27,7 +27,7 @@
   <change-notes><![CDATA[
     <ul>
       <li>
-          Ability to open a commit from the VCS log tool window <a href="https://github.com/ben-gibson/jetbrains-open-in-git-host/issues/22">#22</a>.
+          Fixed: "Open in... Git Host" action works also from editor (It uses the line number from the current editor) <a href="https://github.com/ben-gibson/jetbrains-open-in-git-host/issues/24">#24</a> (PR by markiewb)
```

Comment:
Could this be updated to something like Fixed: Select target action now uses the correct line number when used from the editor.